### ISSUE TITLE
[Bug] the issue video stops and starts buffering at 1:09

### ISSUE THREAD

**Issue by KhangAsassin** [cuynu/ytvancedx#228]
PLEASE FILL OUT THIS FORM, DO NOT CREATE ISSUE WITHOUT THIS FORM, OTHERWISE, ISSUE WILL CLOSE WITHOUT ANY NOTICE OR CONVERT TO DISCUSSION.

Environment
===========
Platform:Android
OS Version:7.0
YouTube Vanced Version:17.39.35
Vanced MicroG:0.2.28.232516
Device:Redmi note 4

Description
===========
When i play any random video on this app, it plays it normally as expected but after 50 seconds it stops playing and gose in loading.. maybe its a buffering issue so i tried following things.

Steps to reproduce
==================
1.Open application
2.Play any video which i above 1 min
3.seet it for around 60 sec, and then you should face loading black screen and video will be stuck over there

Expected result
===============
Video should be able to play smoothly

Actual result
=============

Screenshots / Videos / Logs
===========================

Notes
=====

**Comment by FirdausRoblox031** [comment]
Well, I did need to enable protobuf spoof. But it's still loading... Damn it.

**Comment by OleShes** [comment]
I also ran into this problem. Try to disable "Enable protobuf spoof" option on Vanced settings -> Video. And buffering problem should fix.

**Comment by LePhenix47** [comment]
Thanks! @OleShes

**Comment by OleShes** [comment]
@LePhenix47 
Maybe you're using an old version of YouTube Vanced? I updated from 18.02.33 to 18.21.34 and it helped)

**Comment by bg17aw** [comment]
I also have this problem, tried deleting cache and data for MicroG and Vanced but this did not helped!

**Comment by VancedHelp** [comment]
The solution is here my friends. I've searched for a solution during an hour and finally i've stumble upon this. itS easy and it works for me. Hopefully for you too. https://www.youtube.com/watch?v=w91ObivxSI8&ab_channel=Menosh

**Comment by LePhenix47** [comment]
> @LePhenix47 
> Maybe you're using an old version of YouTube Vanced? I updated from 18.02.33 to 18.21.34 and it helped)

You're absolutely right! Even though I initially downloaded a new version of YT Vanced, it seems the app didn't update as expected.

However, I uninstalled the previous version before reinstalling it fresh. The great news is that after doing so, disabling the protobuf spoof turned out to be the key to resolving the issue!

I'm really grateful for the guidance and support from both homies @OleShes and @VancedHelp !

**Comment by rkf2778** [comment]
> The solution is here my friends. I've searched for a solution during an hour and finally i've stumble upon this. it's easy and it works for me. Hopefully for you too ! https://www.youtube.com/watch?v=w91ObivxSI8&ab_channel=Menosh

THIS WORKED FOR ME FOR A WHILE. BUT THE PROB CONTINUES EVEN AFTER

**Comment by TheArgentinian** [comment]
Vanced 17.34.36
Android 6
Disabling and enabling spoof setting didn't fix anything.
Cleaning cache and erasing data didn't either.

**Comment by rkf2778** [comment]
> > @LePhenix47
> > Maybe you're using an old version of YouTube Vanced? I updated from 18.02.33 to 18.21.34 and it helped)
> 
> You're absolutely right! Even though I initially downloaded a new version of YT Vanced, it seems the app didn't update as expected.
> 
> However, I uninstalled the previous version before reinstalling it fresh. The great news is that after doing so, disabling the protobuf spoof turned out to be the key to resolving the issue!
> 
> I'm really grateful for the guidance and support from both homies @OleShes and @VancedHelp !

Could you please share the link for the version ?

**Comment by VancedHelp** [comment]
Just to let you know in case it could help my youtube vanced version is 18.02.33 the micro G is 0.2.27.231413 and I'm running android 9 on my phone.

**Comment by VancedHelp** [comment]
I had probably talk too fast. The problem came back. Then I've seen it was enable again so I disable spoof settings and it cuts around 1:41 after the video starts. I guess only a new update could solve the problem. I f someone find the solution, thanks in advance.

**Comment by elhaddajiOtmane** [comment]
> @LePhenix47 
> Maybe you're using an old version of YouTube Vanced? I updated from 18.02.33 to 18.21.34 and it helped)

I have the same issue on version 18.21.34 can u tell me what's u vanced micro g version

**Comment by ibeXedi** [comment]
Hello guys.
If you could watch the pinned issue that would be nice.

When you are on the latest versions of microG and YouTube vanced but still suffer loading videos with buffer fix on or off it seems that Google did smth to your account. 

As long as there is no further update from vanced you need to you the inkognito mode. Than it works fine.

**Comment by ibeXedi** [comment]
For me it seems like a Google/YouTube issue.
Im on the latest microG version and Vanced version and with or without buffer fix it stops working after some time.
The funny thing is that even with the official YouTube App the same thing happens.
Maybe we just need to wait until YouTube fixes there stuff.

**Comment by BioCookieYT** [comment]
I had the same issue in the normal YouTube app. Try switching to a other account, it works for me now. I think Google blacklisted some accounts...

**Comment by TheArgentinian** [comment]
I don't think it's account related. I clear the data of Vanced and used it without login in and buffer issue is still there. We need more confirmation. Maybe try with a relative account that never used a modded app?

**Comment by toysnic** [comment]
Same problem here with vanced 18.21.34.
but for me no problem with official YouTube app.
I tried to uncheck protocol buffer but not solve the problem.

**Comment by ibeXedi** [comment]
I just switched the account in the vanced app and it instantly worked again. ¯\_(ツ)_/¯

**Comment by bhavukp** [comment]
I was having the buffering issue yesterday with both vanced and revanced latest apk's and even on the official YouTube app but today the official app is working okay.

**Comment by OleShes** [comment]
@elhaddajiOtmane 
My version of Vanced MicroG is 0.2.24.220220. Everything has been working fine for me for the last 6 hours

**Comment by LePhenix47** [comment]
It still doesn't work, despite having the latest version of MicroG (v 0.2.24.220220) and old YouTube Vanced (v 18.21.34), with protobuf enabled or disabled the issue still persists

The video buffers around the 1 minute mark

**Comment by OleShes** [comment]
Oh, I forgot... I also turned on battery optimization for Vanced MicroG

**Comment by ibeXedi** [comment]
> It still doesn't work, despite having the latest version of MicroG (v 0.2.24.220220) and old YouTube Vanced (v 18.21.34), with protobuf enabled or disabled the issue still persists
> 
> The video buffers around the 1 minute mark

The latest version microG is 0.2.28.233013.
Or is that OS based?

**Comment by LJGJM** [comment]
i have the same issue at 1:28

**Comment by zarigata** [comment]
> I also ran into this problem. Try to disable "Enable protobuf spoof" option on Vanced settings -> Video. And buffering problem should fix.

I dont even have this on my version....

**Comment by donjonne** [comment]
hei guys i found a solution: (this one does not use saved vanced account)
1.  delete vanced manager and youtube vanced.
2.  on phone settings>app>youtube(normal)>storage
clear cache and delete storage
3.  create a new account for your normal youtube account if you dont have more than one account linked. 
4.  sign in to other account or new account inside the youtube(normal) app
5. go to phone settings and app>youtube(normal) FORCE STOP AND DISABLE
6.  Install fresh youtube vanced apk 18.02.33 black universal hotfix only 
7.   open vanced app and on vanced settings>video> enable spoof.  app will reset.   

Enjoy:  if this works for you please consider buying me a beer through paypal:  johnnybravophone.dy@gmail.com
it will only take 3 minutes of your time.  thanks.

**Comment by VancedHelp** [comment]
I've been to this site https://vancedmanager.org/download and install the latest micro G and youtube Vanced wich was released yesterday to fix the buffering issue. At first the problem was still there even before log in to my account but it seems to disappear as soon as I turned on battery optimization for Vanced MicroG. But the problem started again around 3 minutes after the video was playing. And now it stops playing between 1:07 and 1:36. What did I miss ? it's driving me nuts.

**Comment by donjonne** [comment]
> I've been to this site https://vancedmanager.org/download and install the latest micro G and youtube Vanced wich was released yesterday to fix the buffering issue. At first the problem was still there even before log in to my account but it seems to disappear as soon as I turned on battery optimization for Vanced MicroG. But the problem started again around 3 minutes after the video was playing. And now it stops playing between 1:07 and 1:36. What did I miss ? it's driving me nuts.

tried my method?

**Comment by VancedHelp** [comment]
> > I've been to this site https://vancedmanager.org/download and install the latest micro G and youtube Vanced wich was released yesterday to fix the buffering issue. At first the problem was still there even before log in to my account but it seems to disappear as soon as I turned on battery optimization for Vanced MicroG. But the problem started again around 3 minutes after the video was playing. And now it stops playing between 1:07 and 1:36. What did I miss ? it's driving me nuts.
> 
> tried my method?

I will. I've tried first the latest vanced version wich normally solve the problem but since it's exactly the same as before I'll try. It's kind of weird that some tricks do perfectly works for others and not for me.

**Comment by ibeXedi** [comment]
Unfortunatly the only working fix I found was to install revanced and deinstall vanced + :/

https://revanced.net/

**Comment by Lord-Memester** [comment]
I just started encountering this problem today. Not sure why it happened earlier for others.

**Comment by cuynu** [comment]
redownload vanced rvx 18.21.34 black from archive repo and install, it should works. i cant release my Vanced+ as it not stable yet aswell i also getting mentioned from [YT-Advanced](https://github.com/YT-Advanced) github user, an legacy continuation of RVX.

**Comment by TheArgentinian** [comment]
so everyone using old Android devices that can't install the latest version is screwed? :(

**Comment by rahulmahawar06** [comment]
It's not working

**Comment by KindredOther** [comment]
> hei guys i found a solution: (this one does not use saved vanced account)
> 1.  delete vanced manager and youtube vanced.
> 2.  on phone settings>app>youtube(normal)>storage
> clear cache and delete storage
> 3.  create a new account for your normal youtube account if you dont have more than one account linked. 
> 4.  sign in to other account or new account inside the youtube(normal) app
> 5. go to phone settings and app>youtube(normal) FORCE STOP AND DISABLE
> 6.  Install fresh youtube vanced apk 18.02.33 black universal hotfix only 
> 7.   open vanced app and on vanced settings>video> enable spoof.  app will reset.   
> 
> Enjoy:  if this works for you please consider buying me a beer through paypal:  johnnybravophone.dy@gmail.com
> it will only take 3 minutes of your time.  thanks.

This works!

**Comment by donjonne** [comment]
hei guys while there is an official update, the method i suggested still works.
1. clear cache and delete data on vanced
2. keep on making new youtube accounts or login to a different rarely used account on your normal youtube 
3. open vanced again and it should work

i dont know if google is blacklisting emails or whatever but this method resets it and it works again 18.0.2.33

**Comment by TheArgentinian** [comment]
This has nothing to do with accounts! The official app and smarttube for Android TV work just fine with the same account.

**Comment by donjonne** [comment]
> This has nothing to do with accounts! The official app and smarttube for Android TV work just fine with the same account.

official account?
you mean youtube?

**Comment by N3kosama66** [comment]
I have been trying all that have been sugested here. But all of them not working...

**Comment by VancedHelp** [comment]
Hi all I finally found a solution that works for me since few hours. Thanks to @cuynu  for giving us the solution. So first download and install the lastest micro-G here [https://github.com/cuynu/ytvancedx?fbclid=IwAR1CCV2AdmJbcwUHfSBIaeRf8zo5imbGR6c13Cga4Gbbp2oFha90lmCfhsI](url) then download and install the latest Youtube Vanced RVX here [https://github.com/cuynu/archive/releases/tag/18.21.34?fbclid=IwAR25p5At_VhYAy_iBenu6ylLWctwBB5jv85FFVLcFPLc-ysZE20hCgI_YBo](url) . I add some screenshots to help you spot what I'm talking about. I hope it will work for you too. Let me know.
![367572429_938778593879991_1105199814721447769_n](https://github.com/cuynu/ytvancedx/assets/142058807/3b414057-e9a8-4585-a495-49dc772d699a)
![367632317_1493297144818058_6451182351140020024_n](https://github.com/cuynu/ytvancedx/assets/142058807/3c0cdc40-505f-49ad-a779-13fae9e707fc)

**Comment by Lord-Memester** [comment]
@VancedHelp that worked for me, thanks. I didn't even have to update micro-G, the updated vanced rvx file (i picked black) seems to have done the trick, and it only needed to update.

**Comment by TheJohan8** [comment]
It still doesn't work even on normal youtube app.
i tried clearing cache, uninstalling updates for normal yt app, reinstalling vanced, creating new account, this method @cuynu and @donjonne described and nothing worked.

**Comment by donjonne** [comment]
> It still doesn't work even on normal youtube app. i tried clearing cache, uninstalling updates for normal yt app, reinstalling vanced, creating new account, this method @cuynu and @donjonne described and nothing worked.

quick test and the latest release edited fix works
https://github.com/cuynu/archive/releases/tag/18.21.34

the 129 mb file
18.21.34 black universal

**Comment by VancedHelp** [comment]
After 24 hours it still work perfectly. The only downside for me is that I see ads not inside the video but whenever I scroll on the homescreen or after a search some ads appears. Also under the section comment when I play a video. I've been on the vanced settings but the only tweak I see is enable or disable ads inside video wich is already enable. If someone knows how to get rid of it, thanks in advance.

**Comment by TheJohan8** [comment]
@donjonne 

 **works for me thank you**

ps.    I just simply checked out of curiosity normal youtube and still doesn't work but that's not my problem.

**Comment by Lord-Memester** [comment]
> @VancedHelp that worked for me, thanks. I didn't even have to update micro-G, the updated vanced rvx file (i picked black) seems to have done the trick, and it only needed to update.
> 
> I should mention that the normal YouTube app worked for me though, so it's possible that's a key piece of information that I left out.

Update: I had protobuff spoof off when I did this at first, and it was working fine. However, a few hours ago, I encountered a video that froze and buffered at about 40-something seconds in. Upon turning the spoof on, it seems to be working again. I admit I jumped rather quickly to that fix, so it may not have been necessary at all and something as simple as closing and reopening the app may have worked, but the refresh and restart caused by enabling spoof doesn't tell me which specifically did the trick. I'm not particularly interested in fiddling with it given that it currently works. I'll update my experience if anything changes.
